OnlineTradePlatform/OnlineTradePlatform.Application/IServices/IAdService.cs

```diff
@@ -15,6 +17,6 @@ public interface IAdService
 
         Task<PagedList<AdDto>> GetAdPageAsync(PageParameters pageParameters, Expression<Func<Ad, bool>> predicate, CancellationToken cancellationToken);
```

Comment:
change predicate to other parameters, that you will use for filtering, string or int type.